### PR TITLE
Add search bar design pattern

### PR DESCRIPTION
**Pull request by daileytj**
Fixes PXBLUE-1325.

#### Changes proposed in this Pull Request:
- Add search bar design pattern

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -21,15 +22,15 @@ export const NAV_ITEMS: DrawerItem[] = [
...
-            {
-                title: 'Collapsible',
-                path: 'collapsible',
-                component: PlaceholderComponent,
-            },
+            // {
+            //     title: 'Collapsible',
+            //     path: 'collapsible',
+            //     component: PlaceholderComponent,
+            // },
```
Comment (joebochill) [inline src/app/app-routing.module.ts:29]: No love for the collapsible appbar?

**Review by joebochill — CHANGES_REQUESTED** [review]
Code looks reasonable, but running it on a device has some weird behavior. I'm unable to scroll the list vertically, and I can scroll it horizontally to reveal a search bar:
image

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (daileytj) [reply src/app/app-routing.module.ts:29]: > Code looks reasonable, but running it on a device has some weird behavior. I'm unable to scroll the list vertically, and I can scroll it horizontally to reveal a search bar:
> image

Turns out this is actually a bug with the angular implementation as well. Ill look into trying to fix this.

Comment (joebochill) [reply src/app/app-routing.module.ts:29]: I logged an issue for the lists not scrolling vertically: https://github.com/pxblue/ionic-design-patterns/issues/6

**Comment by emclaug2** [comment]
The searchbar is working fine for me now, but when I scroll down the appbar isn't pinned to the top of the screen.

**Comment by joebochill** [comment]
> The searchbar is working fine for me now, but when I scroll down the appbar isn't pinned to the top of the screen.

We've got a followup issue to deal with the wonky scrolling: #6

**Review by joebochill — APPROVED** [review]

**Pull request merged**